xrmocap/core/evaluation/metrics.py

```diff
@@ -0,0 +1,200 @@
+                continue
+
+            dist = vectorize_distance(f_gt_kps3d[np.newaxis], f_pred_kps3d)
+            f_pred_kps3d = f_pred_kps3d[np.argmin(dist[0])]
+
+            for i, limb in enumerate(limbs):
+                start_point, end_point = limb
+                if check_limb_is_correct(f_pred_kps3d[start_point],
+                                         f_pred_kps3d[end_point],
+                                         f_gt_kps3d[start_point],
+                                         f_gt_kps3d[end_point]):
+                    check_result[idx, gt_kps3d_idx, i] = 1
+                    accuracy_cnt += 1
+                else:
+                    check_result[idx, gt_kps3d_idx, i] = -1
+                    error_cnt += 1
+            gt_hip = (f_gt_kps3d[2] + f_gt_kps3d[3]) / 2
+            pred_hip = (f_pred_kps3d[2] + f_pred_kps3d[3]) / 2
+            if check_limb_is_correct(pred_hip, f_pred_kps3d[12], gt_hip,
+                                     f_gt_kps3d[12]):
+                check_result[idx, gt_kps3d_idx, -1] = 1
+                accuracy_cnt += 1
+            else:
+                check_result[idx, gt_kps3d_idx, -1] = -1
+                error_cnt += 1
+    bone_group = dict([('Head', np.array([8])), ('Torso', np.array([9])),
+                       ('Upper arms', np.array([5, 6])),
+                       ('Lower arms', np.array([4, 7])),
+                       ('Upper legs', np.array([1, 2])),
+                       ('Lower legs', np.array([0, 3]))])
+
+    person_wise_avg = np.sum(
+        check_result > 0, axis=(0, 2)) / np.sum(
+            np.abs(check_result), axis=(0, 2))
+
+    bone_wise_result = dict()
+    bone_person_wise_result = dict()
+    for k, v in bone_group.items():
+        bone_wise_result[k] = np.sum(check_result[:, :, v] > 0) / np.sum(
+            np.abs(check_result[:, :, v]))
+        bone_person_wise_result[k] = np.sum(
+            check_result[:, :, v] > 0, axis=(0, 2)) / np.sum(
+                np.abs(check_result[:, :, v]), axis=(0, 2))
+
+    tb = PrettyTable()
+    tb.field_names = ['Bone Group'] + [
+        f'Actor {i}' for i in range(bone_person_wise_result['Head'].shape[0])
+    ] + ['Average']
+    list_tb = [tb.field_names]
+    for k, v in bone_person_wise_result.items():
+        this_row = [k] + [np.char.mod('%.4f', i) for i in v
+                          ] + [np.char.mod('%.4f',
+                                           np.sum(v) / len(v))]
+        list_tb.append([
+            float(i) if isinstance(i, type(np.array([]))) else i
+            for i in this_row
+        ])
+        tb.add_row(this_row)
+    this_row = ['Total'] + [
+        np.char.mod('%.4f', i) for i in person_wise_avg
+    ] + [np.char.mod('%.4f',
+                     np.sum(person_wise_avg) / len(person_wise_avg))]
+    tb.add_row(this_row)
+    list_tb.append([
+        float(i) if isinstance(i, type(np.array([]))) else i for i in this_row
+    ])
+    if dump_dir:
```

Comment:
Same as https://github.com/openxrlab/xrmocap/pull/16/files#r980693268. `def calc_limbs_accuracy` does caculating, not saving files.